### ISSUE TITLE
Not possible to open the camera settings

### ISSUE THREAD

**Issue by chb9** [cmauri/eviacam#10]
With eviacam 2.1.2 (master branch) on Debian 9.1 (stable, XFCE) it is not possible to open the camera settings. (see screenshot)

![camera](https://user-images.githubusercontent.com/30769158/31315234-54cc6e08-ac14-11e7-999d-f09bcd148d97.png)

**Comment by cmauri** [comment]
Right. Since commit 765a2bff550db20584dee275b9af4d815db638da eViacam allows you only to use the OpenCV based camera capture API which lacks support for configuration dialog. I removed support for custom V4L2 capture due to the fact that does not support JPEG format which is the only available for some webcam models.

To be able to configure the camera parameters you have 2 options:

1) Revert such a commit and recompile again. That will be fine if your camera provides capture formats other than JPEG

2) Open an external program to adjust camera settings. guvcview works for me, when started it complains, but eventually works

**Comment by chb9** [comment]
ok, thank you!
